Pal/src/host/Linux-SGX/signer/pal-sgx-sign

```diff
@@ -352,14 +352,13 @@ def get_memory_areas(manifest, attr, args):
 
 def populate_memory_areas(manifest, attr, areas):
     populating = attr['enclave_size']
-    heap_min = attr['heap_min']
```

Comment:
Update all the manifest template in the repository if we decide to officially deprecate `sgx.heap_min`.

Reply:
Do we think heap_min is a useful parameter in any of our workloads seen so far?

Reply:
So far, no. Originally it was for determining whether the executable is position-independent or not (sgx.heap_min = 0x400000 means position-dependence). Now the signer can tell whether the executable is position-independent by reading the ELF header. Removing it should be perfectly fine.

Reply:
Ok.  I only found it in the openjdk template.  Removed.